ui-web/sdk/src/index.ts

```diff
@@ -74,17 +74,21 @@ export async function launchRedirect(launchUrl: string) {
 /**
  * Launches the user through a popup towards any of the Trinsic launch URLs (applies to Widget, Hosted and Advanced Sessions).
  * @param getLaunchUrl - A promise you give us where you retrieve the launch from your backend. This is a promise because of browser restrictions in Safari where popups need to be created, opened and directed to the final domain in the same event loop.
+ * @param loadingDomain - The domain of the loading page that will be used to load the launch url. The domain will first need to be added to the dashboard. Optional, otherwise will load from api.trinsic.id
```

Comment:
```suggestion
 * @param loadingDomain - The domain of the loading page that will be used to load the launch url. This only applies if you are using domain white labelling for the session. The domain will first need to be added to the dashboard. Optional, otherwise will load from api.trinsic.id
```